metrics/collectors/images/images_collector.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021 Canonical Ltd
+
+from metrics.lib.basemetric import Metric
+from launchpadlib.launchpad import Launchpad
+
+import requests, datetime, tempfile
+
+DESKTOP_CURRENT_DAILY = "http://cdimage.ubuntu.com/daily-live/current/%s-desktop-%s.iso"
+DESKTOP_LTS_DAILY = "http://cdimage.ubuntu.com/%s/daily-live/current/%s-desktop-%s.iso"
```

Comment:
Could we maybe collect for `pending` as well as `current` (having that as a tag maybe)? It could be useful to know what's going on with the pending image for example even if promotion is not happening. Like the age there could be telling you that the image is failing to build.

Reply:
The new revision handle current and pending images